src/components/ValueInput/ValueInput.js

```diff
@@ -251,7 +251,7 @@ export const ValueInputComponent = (props: Props) => {
     if (onFormValid) {
       onFormValid(!errorMessage);
     }
-  }, [value, assetData]);
+  }, [value, assetData, customBalances]);
```

Comment:
Validity of form depends on asset balances, because we check for insufficient amount

Here are steps to reproduce a bug:
- remove `customBalances` from dependencies
- select eth, type some small amount, but large enough to cause "insufficient eth" error and disable "Next" button
- change token to dai or usdc 
- dai or usdc is way cheaper than eth and doesn't cause "insufficient funds" error
- the "Next" button should be unblocked, but isn't o.o

why? I think it goes like this:
- user selects dai
- `selectedAsset` changes to DAI
- `assetData` in ValueInput changes to DAI
- effects are triggered, form is not valid, because we 0 balance of dai at first
- we recalculate the balance, set it, but the onFormValid is not triggered, form is still invalid and it should be valid now

Reply:
But `errorMessage` is recalculated on every render anyway, so it's always up to date. Would there be a problem with setting it as the sole dependency? Then there's no need to know the implementation details of `getErrorMessage`, and it should get rid of these kind of bugs.